docs/guide.md

```diff
@@ -727,7 +729,7 @@ const p = new Page({ name: 'mongodb.org' });
 console.log(p.id); // undefined
 ```
 
-<h2 id="_id"><a href="#_id">option: _id</a></h2>
+## option: _id {#_id}
```

Comment:
duplicate id with `## Ids {#_id}`, maybe we should change one of them?

Reply:
Good catch, fixed in edd94fc78ea9edd506a931f39489c571050cb91f